package.json

```diff
@@ -169,6 +170,7 @@
     "raw-loader": "^4.0.2",
     "sass": "^1.62.0",
     "sass-loader": "^10.4.1",
+    "sharp": "0.31.3",
```

Comment:
Why is sharp here?

Reply:
let me check again if I remove that dep

previously I got this error
```
sharp@0.31.3/node_modules/sharp install: gyp ERR! cwd /home/runner/work/nft-gallery/nft-gallery/node_modules/.pnpm/sharp@0.31.3/node_modules/sharp
```

Reply:
seems ok now